usaspending_api/conftest_helpers.py

```diff
@@ -43,12 +43,17 @@ def update_index(self):
 
     def _add_contents(self):
         """
-        Get all of the transactions presented in the view and
-        stuff them into the Elasticsearch index.
+        Get all of the transactions presented in the view and stuff them into the Elasticsearch index.
+        The view is only needed to load the transactions into Elasticsearch so it is dropped after each use.
         """
+        transaction_delta_view_sql = open(
+            str(settings.APP_DIR / "database_scripts" / "etl" / "transaction_delta_view.sql"), "r"
+        ).read()
         with connection.cursor() as cursor:
-            cursor.execute("SELECT * FROM {}".format(settings.ES_TRANSACTIONS_ETL_VIEW_NAME))
+            cursor.execute(transaction_delta_view_sql)
+            cursor.execute(f"SELECT * FROM {settings.ES_TRANSACTIONS_ETL_VIEW_NAME};")
             transactions = ordered_dictionary_fetcher(cursor)
+            cursor.execute(f"DROP VIEW IF EXISTS {settings.ES_TRANSACTIONS_ETL_VIEW_NAME};")
```

Comment:
Why `IF EXISTS`? Do we really think this is going to drop? Harmless I suppose, but this could be a way to fail loudly...

Reply:
Moved it around a couple of times and didn't think to take that out. Will edit on the next set of changes.